src/docfx/build/redirection/RedirectionItem.cs

```diff
@@ -19,4 +19,8 @@ internal class RedirectionItem
     public SourceInfo<string> RedirectUrl { get; set; } = new SourceInfo<string>("");
 
     public bool RedirectDocumentId { get; set; }
+
+    // the path of redirection configuration file
```

Comment:
Leave `source_url` for redirections or other generated files as `undefined|null|unassigned` because there is no such physical file. If this point to the configuration, then `source_url` would not agree with the `source_path` property which is both confusing and incorrect.